src/plugins/telemetryTable/components/table.vue

```diff
@@ -62,8 +65,38 @@
                 {{ paused ? 'Play' : 'Pause' }}
             </span>
         </button>
+
         <slot name="buttons"></slot>
     </div>
+    <!-- main controlbar end -->
+
+    <!-- alternate controlbar start -->
+    <div v-if="alternateControlBar.enable && markedRows.length"
+         class="c-table-control-bar c-control-bar"
+    >
+        <div class="c-control-bar__label">
+            {{ markedRows.length > 1 ? `${markedRows.length} ${alternateControlBar.rowNamePlural} selected`: `${markedRows.length} ${alternateControlBar.rowName} selected` }}
+        </div>
+
+        <toggle-switch
+            id="show-filtered-rows-toggle"
```

Comment:
How is it guaranteed that the id is unique?

Reply:
In the table context, there is only one toggle switch and is only in the show filtered rows context, it will never collide with another one